src/Okta.Sdk/Generated/LogClient.Generated.cs

```diff
@@ -1,38 +0,0 @@
-// <copyright file="LogClient.Generated.cs" company="Okta, Inc">
```

Comment:
Did `LogClientInfo.Generated.cs` get created?

Reply:
Yes, it was created as part of the previous "partial" branch dedicated to this matter.